apps/editors/helpers.py

```diff
@@ -476,8 +478,13 @@ def get_actions(self, request, addon):
         labels, details = self._review_actions()
 
         actions = SortedDict()
-        if not addon.admin_review or acl.action_allowed(
-                request, 'ReviewerAdminTools', 'View'):
+        if ((not addon.admin_review or acl.action_allowed(request,
+             'ReviewerAdminTools', 'View'))
+            and (not GroupUser.objects.filter(group=Group.objects.get(
```

Comment:
this new if branch needs a test otherwise someone else might break it accidentally. Maybe you can copy something from https://github.com/wagnerand/olympia/blob/1443/apps/editors/tests/test_views.py